swift/ql/lib/codeql/swift/generated/Children.qll

```diff
@@ -0,0 +1,210 @@
+// generated by codegen/codegen.py
+import codeql.swift.elements
+
+cached
+Element getAChild(Element e) {
+  not exists(e.getResolveStep()) and
```

Comment:
Do we need this condition? To use this in CFG construction we'd need to get rid of this condition since CFG construction uses the fully converted expression everywhere (i.e., it doesn't skip "hidden" AST nodes).

Reply:
For the record, what we just discussed online: it seems we need `getAChild/getParent` to ignore hiding of AST nodes. I will rename them to `getAnImmediateChild` and `getImmediateParent` and ignore resolution by using db predicates